alpine/entrypoint.sh

```diff
@@ -90,6 +90,11 @@ fi
 
 if [[ $KUBERNETES ]]; then
     cp /opt/datadog-agent/agent/conf.d/kubernetes.yaml.example /opt/datadog-agent/agent/conf.d/kubernetes.yaml
+
+    # enable event collector
+    if [[ $KUBERNETES_COLLECT_EVENTS ]]; then
+        sed -i -e "s@# collect_events: False@collect_events: True@" /etc/dd-agent/conf.d/kubernetes.yaml
```

Comment:
:itsatrap: config files are in `/opt/datadog-agent/agent/conf.d/` on alpine (it's just a source install that we move there). See above